lib/chef/resource/route.rb

```diff
@@ -27,20 +27,26 @@ class Route < Chef::Resource
 
       description "Use the route resource to manage the system routing table in a Linux environment."
 
-      property :target, String, identity: true, name_property: true
-      property :comment, [String, nil]
-      property :metric, [Integer, nil]
-      property :netmask, [String, nil]
-      property :gateway, [String, nil]
-      property :device, [String, nil], desired_state: false # Has a partial default in the provider of eth0.
-      property :route_type, [:host, :net], default: :host, coerce: proc { |x| x.to_sym }, desired_state: false
-
-      # I can find no evidence of these properties actually being used by Chef. NK 2017-04-11
-      property :networking, [String, nil], desired_state: false
-      property :networking_ipv6, [String, nil], desired_state: false
-      property :hostname, [String, nil], desired_state: false
-      property :domainname, [String, nil], desired_state: false
-      property :domain, [String, nil], desired_state: false
```

Comment:
These are all part of the `route` public API and can't be removed until chef 15

Reply:
That's totally fine. This can wait till we branch off chef-14.